optuna/samplers/_partial_fixed.py

```diff
@@ -92,16 +92,17 @@ def sample_independent(
     ) -> Any:
 
         # If param_name isn't in self._fixed_params.keys(), param_value is set to None.
-        param_value = self._fixed_params.get(param_name)
-
-        if param_value is None:
+        # And if param_value is `None` itself it should be `None`
+        if param_name not in self._fixed_params.keys():
             # Unfixed params are sampled here.
             return self._base_sampler.sample_independent(
                 study, trial, param_name, param_distribution
             )
         else:
             # Fixed params are sampled here.
             # Check if a parameter value is contained in the range of this distribution.
+            param_value = self._fixed_params.get(param_name)
```

Comment:
(nitpicky) `self._fixed_params[param_name]` seems to be better because it represents we can assume the key is in the dict.

```suggestion
            param_value = self._fixed_params[param_name]
```

Reply:
I updated it, as you suggest. :)
Thank you for the review!